cmake/modules/FindPortAudio.cmake

```diff
@@ -89,6 +91,14 @@ if(PortAudio_FOUND)
     )
     is_static_library(PortAudio_IS_STATIC PortAudio::PortAudio)
     if(PortAudio_IS_STATIC)
+      if(PortAudio_ALSA_H)
```

Comment:
We should probably check if we need this, a few lines above the file states that this was a workaround to provide compile-time defines for checking whether ALSA exists, so we might not want to rely on `PortAudio_ALSA_H`.

Is it safe to simply link ALSA whenever we find it?

Reply:
Yes, we do not "link" it. We just tell the linker "Here is something that you may want to link". So you solution should be safe in any case.